sdk/eventhub/azure-eventhub/out/azure/eventhub/aio/_consumer_client_async.pyi

```diff
@@ -0,0 +1,22 @@
+class EventHubConsumerClient(ClientBaseAsync):
+
+    #...
+
+    async def receive_batch(
+            self, on_event_batch: Callable[[PartitionContext, List[EventData]], Awaitable[None]],
+            *,
+            max_batch_size: int=...,
+            max_wait_time: float=...,
+            callback_without_event: bool=...,
```

Comment:
This parameter name looks odd.

Reply:
Changed to `enable_callback_when_no_event`. I'm open to better names.